measure.js

```diff
@@ -41,7 +41,8 @@ var ids = ["1348773b", "6808dfab", "0717b399", "a9223e93", "cb383bfd"];
 async function executeCycle() {
   var cycles = 200;
   var ets = [];
-
+  var unnecesary = [];
+  var unnecesary2 = [];
   for (let i = 0; i < cycles; i++) {
     for (const value of ids) {
       const et = await getUserInfo(value);
```

Comment:
Ensure that the 'getUserInfo' function is properly handling errors and logging relevant information